github/pull_request.rb

```diff
@@ -105,6 +113,15 @@ def check_commits_style
     end
   end
 
+  def add_issue_links
+    if new? && issue_numbers.any?
+      message = issue_numbers.inject("\nIssues:") do |msg, issue_number|
+        msg + " [##{issue_number}](http://projects.theforeman.org/issues/#{issue_number})"
+      end
+      append_pull_request_body(message)
```

Comment:
Correct, it adds the issue links to the description of the PR on a new line. One big advantage is that no new notifications will be sent to subscribers on that PR/project. It looks like this:

> The original description the user makes
> Issues: [#19023](http://projects.theforeman.org/issues/19023) [#14611](http://projects.theforeman.org/issues/14611)